evennia/contrib/utils/database_backup/README.md

```diff
@@ -0,0 +1,69 @@
+# Database Backup Scheduler
+
+Contribution by helpme (2024)
+
+This module schedules backups in-game, which saves a copy of your database to your game's `server/backups` folder. Database backups are *not* automatically uploaded to any cloud service, it is left to you to decide what to do with them (i.e. pushed to git, uploaded to the ether, downloaded to a hard drive).
+
+Backups can take place at any time. Restoring the game world from backup takes place during downtime, as documented below.
+
+Currently, the sqlite3 (the evennia default) and postgresql databases are supported. Others are welcome to add more.
+
+## Installation
+
+This utility adds the `backup` command. Import the module into your commands and add it to your command set to make it available.
+
+In `mygame/commands/default_cmdsets.py`:
+
+```python
+...
+from evennia.contrib.utils.database_backup import DbCmdSet   # <---
+
+class CharacterCmdset(default_cmds.Character_CmdSet):
+    ...
+    def at_cmdset_creation(self):
+        ...
+        self.add(DbCmdSet)  # <---
+
+```
+
+Then `reload` to make the `backup` command available.
+
+## Permissions
+
+By default, the backup command is only available to those with Developer permissions and higher. You can change this by overriding the command and setting its locks from "cmd:pperm(Developer)" to the lock of your choice.
+
+## Settings Used
+
+This utility uses the settings.DATABASES dictionary.
+
+## Restoration
+
+Remember to `evennia stop` before restoring your db.
+
+### Restoring sqlite3 (.db3)
+
+* Copy the database backup you want to restore from back into the `server/` directory
+* Rename the database backup to `evennia.db3` if you have not modified `settings.DATABASES`, otherwise whatever name is in your `settings.DATABASES` dictionary.
```

Comment:
If you don't know that the sqlite3 file is named `evennia.db3`, this instruction becomes a little confusing. I think it would be clearer if this was phrased something like
```
By default (unless you changed the name of the file in `settings DATABASES`), the sqlite3 file `mygame/server/evennia.db3`. Copy your backup file over this file to recover your backup.
```